src/main/java/com/google/flourbot/execution/MacroExecutionModuleImplementation.java

```diff
@@ -177,8 +191,14 @@ private String getDate(String pattern) {
     return myDateObj.format(myFormatObj);
   }
 
-  private String selectRandomFromData(List<String> data) {
+  private String selectRandomFromList(List<String> data) {
     Random rand = new Random();
     return data.get(rand.nextInt(data.size()));
   }
+
+  private String selectRandomFromListList(List<List<String>> data) {
```

Comment:
Why are we using a nested list here? What is the outer list holding?

Reply:
It was originally to select a random entry from a sheet, which the big list holds little lists (and the little lists are the rows)